lib/internal/bootstrap/node.js

```diff
@@ -319,6 +319,18 @@ ObjectDefineProperty(process, 'features', {
 }
 
 const { emitWarning, emitWarningSync } = require('internal/process/warning');
+const { getOptionValue } = require('internal/options');
+
+let hasTypeScriptSupport;
+ObjectDefineProperty(process.features, 'typescript', {
+  __proto__: null,
+  get() {
+    return hasTypeScriptSupport ??= !!getOptionValue('--experimental-strip-types');
```

Comment:
With https://github.com/nodejs/node/pull/54283, I think it would be more suitable to return a string here:

- `"strip"` when `--experimental-strip-types` enabled,
- `"transform"` when `--experimental-transform-types` enabled.

Reply:
cloudflared-amd64.pkg: cb3b1ba59a1910e0d74e55ebfef0d64bb1ca642d4f83d060c2b495bcb5be4155
cloudflared-arm64.pkg: 54198dea63f02b8733290773f3b9818a60e0ac844fecdbf3b85b9d63849c8614
cloudflared-darwin-amd64.tgz: 94a792feab15f85ddf01f7a24e4d34a061433073b7a9ace3b3c328672db22a1b
cloudflared-darwin-arm64.tgz: c9ae3a68d4a0a929fe149d7a7e7772209fbe3c41bdcbd729d98cda7d33fdfc54
cloudflared-fips-linux-amd64: fffba42cf46de5ce4b6eb38916dd100b6bd9dd3d5e15f555ca7e5ad8e9337192
cloudflared-fips-linux-amd64.deb: 6d671de82ca756c99a845302f611b41f25ca6d3cb03763266e876a19c292b360
cloudflared-fips-linux-x86_64.rpm: 10de2ff37d80f8699a7b8e0fca49b9c4ff465af58e65774eba3bb69bc5de5642
cloudflared-linux-386: 2860256ecaabdfb9b653712fbacad2b384c84504ea05332d1f4dd91f2a538b6a
cloudflared-linux-386.deb: aef1f1e58337e99a8bf597ddd0c48a638411044a9d94ca49ca6e1e3a2c85d6a5
cloudflared-linux-386.rpm: d4a9a713dd75a12ebd8edd576bb18779a7756a56e1da33c7cbfbbb1d3ff19f2c
cloudflared-linux-aarch64.rpm: 627333275bff0d60c0602738cb00def1a9e481283104fda308fc406d5055cd34
cloudflared-linux-amd64: a0a4c5ea9df1aab96e3dfc41e63d9ee93afff03cc1690cde89f228aa8891f4ea
cloudflared-linux-amd64.deb: 7738cf3cce463574c3cd18b15c1a42c783d0e5b1a1d1f3988f3dd5916c0ea842
cloudflared-linux-arm: 76fbbf5d6504cd1e38af14732b4756c179f3f0462801be6de5c49efdbc799f98
cloudflared-linux-arm.deb: ed3d3eccb32b8a657d54efaa44e66cdb27e48b7ecbe5d812694de207651e60e9
cloudflared-linux-arm.rpm: e72590d1648f4c231625810f4c557eb3f7e632210232398344a12fdf3770c5a2
cloudflared-linux-arm64: d9f70fe26a2e050f22ff9532912a9745c4eac268abb9ad10967fe899773b7475
cloudflared-linux-arm64.deb: 964521c37b45a8016b7ca23223c46747657e6e0e78576bc3ba5247aaf5cc162c
cloudflared-linux-armhf: fe7d5951378c1b8b62b7aa3d96a5b07c0d38098506b4dd8442e7fa27db80bf11
cloudflared-linux-armhf.deb: e971fbb56f5afd230a9929f66273d158598609066cea8525b03c6be6eb31653e
cloudflared-linux-armhf.rpm: 4839bfba3fbb4194904951275a54ca34659c7c77ed97defafa2b5afe65c25537
cloudflared-linux-x86_64.rpm: 0db2edce43b7e662a4d8c6589e52d733f3eca9ccc2d11cff218480520b862e18
cloudflared-windows-386.exe: 9628817bacacf80bb3b3c2826f85bc897ef4d1ddad117d2d56c5c6aa57226f90
cloudflared-windows-386.msi: eb8edbbf8d57b5e40269fd744a28ae089d4054bcb4582824d65ff1d98227dc16
cloudflared-windows-amd64.exe: 38cf822681f0afe89ed6d4b7ee5b2e7fd44880337fcb0d8dd3a1cd9dd8bd83aa
cloudflared-windows-amd64.msi: 763ecdb186c9fd2f5e018ed6362130a4314f478318ed6d6e3f2c7b11490ed0c6

Reply:
pht.pht784